bin/generate_valid_config_file.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+"""Generates a validation configuration file from JSON struct data"""
+
+# Note:  This is written in Python instead of Julia because the Julia YAML
+# package does not support writing (only reading).
+
+import json
+import os
+import sys
+import pdb
+from collections import OrderedDict
+
+
+import yaml
+
+
+POWER_SYSTEM_DESCRIPTOR_FILE = os.path.join(
+    "src",
+    "descriptors",
+    "power_system_structs.json"
+)
+
+def setup_yaml():
+  """ https://stackoverflow.com/a/8661021 """
+  represent_dict_order = lambda self, data:  self.represent_mapping('tag:yaml.org,2002:map', data.items())
+  yaml.add_representer(OrderedDict, represent_dict_order)
+setup_yaml()
```

Comment:
Is `setup_yaml()` being called here? I would move it to the bottom of the file. I would also add a guard:

```
if __name__ == "__main__":
    setup_yaml()
```